src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,181 @@
+package duke;
```

Comment:
Interesting implementation for storing the tasklist, learnt another way of approaching this from your code. I only stored the task list when the bye command was issued but yours was to store every time commands were entered